src/odemis/driver/test/technolution_test.py

```diff
@@ -237,6 +225,8 @@ def test_assembleCalibrationMetadata(self):
         #  dwell_time is decreased. However, for the original maximum dwell time of 1e-4 seconds, this value
         #  needs to be increased on the ASM HW to a value above 9000.
 
+        # FIXME move checks that test the correct calculation of the calibration setpoints to separate test case
```

Comment:
Could you do it now? Maybe just a matter of adjusting the old `test_convertBits()`?

Reply:
It needs a test case for each of the getCalibrationSetpoints on the scanner and descanner. I added them.